recipes/cbcflow/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "cbcflow" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cbcflow-{{ version }}.tar.gz
+  sha256: b70bdfae30568d2f4b89428391e3965b3f298c7572790664a7858d42d1839b71
+
+build:
+  entry_points:
+    - cbcflow_print = cbcflow:cbcflow.print_metadata
+    - cbcflow_pull = cbcflow:cbcflow.pull
+    - cbcflow_update_from_flags = cbcflow:cbcflow.update
+    - cbcflow_update_from_file = cbcflow:cbcflow.from_file
+    - cbcflow_monitor_make = cbcflow:generate_crondor
+    - cbcflow_monitor_run = cbcflow:run_monitor
+    - cbcflow_validate_library = cbcflow:cbcflow.validate_library
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools >=42
+    - setuptools-scm >=3.4.3
+    - wheel
+    - pip
+    - git
+  run:
+    - argcomplete
+    - coverage
+    - gitpython
+    - gwpy
+    - jsondiff
+    - jsonmerge
+    - jsonschema
+    - ligo-gracedb
+    - lscsoft-glue
+    - python-benedict
+    - python >=3.8
+    - pyyaml
+
+test:
+  imports:
+    - cbcflow
+  commands:
+    - pip check
+    - cbcflow_print --help
+    - cbcflow_pull --help
+    - cbcflow_update_from_flags --help
+    - cbcflow_update_from_file --help
+    - cbcflow_monitor_make --help
+    - cbcflow_monitor_run --help
+    - cbcflow_validate_library --help
+  requires:
+    - pip
+
+about:
+  home: https://git.ligo.org/cbc/meta-data
+  summary: A package for enabling CBC analyses
```

Comment:
Since conda-forge is not a LIGO thing, can you give a `description` here as well, to include more context (what's a CBC?).